packages/flutter/test/material/tabs_test.dart

```diff
@@ -2229,7 +2229,6 @@ void main() {
       ),
     );
 
-    expect(tester.hasRunningAnimations, isFalse);
```

Comment:
Why was this removed?

Reply:
Actually, it doesn't need to be removed.  For a while I had a bug (but didn't know it) that had the hover animation still running here.  It's fixed now.